lib/internal/console/constructor.js

```diff
@@ -330,13 +330,13 @@ const consoleMethods = {
     trace(kTraceInstant, kTraceConsoleCategory, `time::${label}`, 0);
   },
 
-  trace(...args) {
+  trace: function trace(...args) {
```

Comment:
Is this strictly needed? I thought the shorthand notation had the same behavior.

Reply:
With the shorthand notation, the function would be called as `obj.property`. It is not possible to access the function itself by only using the property name. That is only possible inside the old `function foo` type.

It would also be possible if this would not be a property but a variable.